source/common/config/filter_json.cc

```diff
@@ -430,5 +430,31 @@ void FilterJson::translateClientSslAuthFilter(
                                       *proto_config.mutable_ip_white_list());
 }
 
+void FilterJson::translateIpTaggingFilterConfig(
```

Comment:
OT, but curious when Lyft will be switching to v2 for LDS?

Reply:
We don't currently use LDS per se anywhere. But to your point the static listeners for our edge proxy are defined in v2 format. So we don't need this to be v1. @ccaraman I would vote to just do v2. Unless you know of a third party that **needs** v1

Reply:
I couldn't find any thirdparty that uses v1. I'll remove it here and submit a subsequent pr to update data plane.

Reply:
Removed mention of v1 in data-plane https://github.com/envoyproxy/data-plane-api/pull/519